### PR TITLE
Add a specific verion for the image gcr.io/kaniko-project/executor

### PR DESCRIPTION
**Pull request by houshengbo**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

**Comment by vdemeester** [comment]
@houshengbo can you rebase against master ?

**Comment by houshengbo** [comment]
sure.

File: docs/taskruns.md [hunk 1]
```diff
@@ -333,11 +333,15 @@ spec:
...
       - name: build-and-push
+<<<<<<< HEAD
         image: gcr.io/kaniko-project/executor
         # specifying DOCKER_CONFIG is required to allow kaniko to detect docker credential
         env:
           - name: "DOCKER_CONFIG"
             value: "/builder/home/.docker/"
+=======
+        image: gcr.io/kaniko-project/executor:v0.9.0
+>>>>>>> 85e787e8... Add a specific verion for the image gcr.io/kaniko-project/executor
```
Comment (bobcatfish) [inline docs/taskruns.md]: i think something went wrong with the merge conflict resolution here @houshengbo ! I think what you need to do is update line 337 with `v0.9.0`, remove `<<<<<<< HEAD` and remove everything from `=======` to `>>>>>>> 85e787e8... Add a specific verion for the image gcr.io/kaniko-project/executor`

Comment (houshengbo) [reply docs/taskruns.md]: oh soory. forgot to resolve it.

**Comment by bobcatfish** [comment]
Thanks @houshengbo !

/lgtm
/meow

**Pull request merged**